src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -321,6 +321,10 @@ public static bool IsInRefContext(this ExpressionSyntax expression)
             => expression.IsParentKind(SyntaxKind.RefExpression) ||
                (expression?.Parent as ArgumentSyntax)?.RefOrOutKeyword.Kind() == SyntaxKind.RefKeyword;
 
+        public static bool IsInInContext(this ExpressionSyntax expression)
+            => expression.IsParentKind(SyntaxKind.RefExpression) ||
```

Comment:
this does not seem to match the name of the exension.  The extension should probably be called "IsInRefExpressionOrInInContext.

Reply:
Will refactor.